### DISCUSSION TITLE
Show miss typed keys while typing?

### DISCUSSION THREAD

**Discussion by bobby-sills** [monkeytypegame/monkeytype#4433] [Ideas] [answered]
I thought this was already a setting, and it might be, but I couldn't find it. The default setting for monkeytype, is that when you make a mistake, it moves the cursor to the next character, and shows the letter that you were supposed to type colored red. What I think would be a useful option, is to be able to change it so that instead of it showing the letter that you were _supposed_ to type, it shows the letter that you actually _did_ type.
Not like this:
![monkeytype mistake](https://github.com/monkeytypegame/monkeytype/assets/76852105/0c457c11-d6f8-407a-95d0-d97c5b67d3d8)

**Comment by fehmer** [comment, marked as answer]
Can be changed in the settings "Indicate typos"

![image](https://github.com/monkeytypegame/monkeytype/assets/3728838/672e978b-ef79-4bab-9105-563d048f3f71)
